src/Umbraco.Web.UI.Client/src/views/components/umb-media-grid.html

```diff
@@ -8,6 +9,7 @@
         ng-class="{'-selected': item.selected, '-file': !item.thumbnail, '-folder': item.isFolder, '-svg': item.extension == 'svg', '-selectable': item.selectable, '-unselectable': !item.selectable}"
     >
         <div>
+                <a ng-if="allowOnClickEdit === 'true'" ng-click="clickEdit(item, $event)" ng-href="" class="icon-edit umb-media-grid__edit"></a>
```

Comment:
Following the rest of the accessibility updates, can you make this a `<button type="button" .. etc></button>` please?

An example update from anchor to button is here: https://github.com/umbraco/Umbraco-CMS/pull/8162/files#diff-6dadd238fd47b0ef06ceec320c1325adR27

Reply:
> Following the rest of the accessibility updates, can you make this a `<button type="button" .. etc></button>` please?

Well this is a strange thing... I didn't add that 'allowonclickedit' link, it's not part of the List View bits I'm adding... looking at the history it was removed in commit 0992bd8 as part of the Media Picker UX Updates - so I think when that has been merged into this branch, the conflict hasn't been resolved correctly, and it's been accidentally reintroduced.

So I've taken the existing HTML from the v8/contrib brand for the umb-media-grid.html to ensure, it's completely correct and up-to-date and so that removes this link, so no need to transform it into a button, as it no longer exists!

Reply:
@marcemarc What I meant.. instead of having actions happen by clicking on a fake link, we should change it into a button. I'll update this one!

Reply:
Aha.. I see what happened now.. indeed, no need to turn it into a button if it's not there any more! Merges get confusing haha!